gno.land/pkg/sdk/vm/errors.go

```diff
@@ -13,16 +13,22 @@ type (
 	InvalidPkgPathError struct{ abciError }
 	InvalidStmtError    struct{ abciError }
 	InvalidExprError    struct{ abciError }
+	InvalidPackageError struct{ abciError }
 )
 
 func (e InvalidPkgPathError) Error() string { return "invalid package path" }
+func (e InvalidPackageError) Error() string { return "package validation failed" }
```

Comment:
Please add a documentation reference in this PR as to what is a "valid package" 🙏